spec/models/outgoing_message_spec.rb

```diff
@@ -143,12 +143,36 @@
     expect(@outgoing_message.get_text_for_indexing).not_to include("foo@bar.com")
   end
 
-  it "should not display email addresses on page" do
-    expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
-  end
+  context "when formatting the message for html display" do
+
+    it "does not display email addresses on page" do
+      expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
+    end
+
+    it "links to help page where email address was" do
+      expect(@outgoing_message.get_body_for_html_display).to include('<a href="/help/officers#mobiles">')
+    end
+
+    it "does not force long lines to wrap" do
+      long_line = "123456789 123456789 123456789 123456789 123456789 123456789 1234"
+      @outgoing_message.body = long_line
+      expect(@outgoing_message.get_body_for_html_display).to include(long_line)
```

Comment:
(but apart from that, and the fact that that only makes sense to recovering mainframe programmers who had to set fixed width layouts for dot matrix printers... \*puts character count ruler back in the drawer\*)